requirements/base.txt

```diff
@@ -27,3 +27,6 @@ typing_extensions~=4.4.0
 regex==2021.9.30
 # NOTE: tzlocal is not a direct dependency of SAM CLI, but pin to 3.0 as 4.0 break appveyor jobs
 tzlocal==3.0
+
+#Adding cfn-lint dependency for SAM validate
+cfn-lint==0.72.2
```

Comment:
does it make sense to pin the patch version considering how frequently cfn-lint gets updated?

```
aws-sam-cli 1.69.0 requires cfn-lint==0.72.2, but you have cfn-lint 0.72.8 which is incompatible.
```